chroma_core/lib/service_config.py

```diff
@@ -652,6 +652,7 @@ def set_nginx_config(self):
             "HTTP_FRONTEND_PORT",
             "HTTPS_FRONTEND_PORT",
             "HTTP_AGENT_PROXY_PASS",
+            "HTTP_AGENT2_PORT",
```

Comment:
I'm not entirely sure this is needed. I can try removing it and rebuilding everything from scratch to make sure it still works.

Reply:
Sounds good.